### ISSUE TITLE
Better Communication Channel for contributors

### ISSUE THREAD

**Issue by Leonardo-Ferreira** [grandnode/grandnode#645]
Hi, I'm a experienced developer that has been working on the field for over 10 years and I would like to contribute to this project.

Right now I'm feeling kinda lost with the project I would like to propose that a slack (free tier) is created so we (the developers who are working on the core of the project, and not on plugins) can chat and hopefully accelerate the time to first contribution of new contributors. This would also be a good place to sync task status and of course bounce ideas for implementation and features!

What you guys think?

If such channel already exists, would you mind sending me a invite? or this is a paid thing?

**Comment by PatrykPorabik** [comment]
Hi Leonardo,

That's fantastic that you want to join to our community. In the case of community channel. To support our customers, we use community forum on our page - https://grandnode.com/boards 
Maybe it will work also for contributors and developers. In the case of tools such as Slack, Gitter, we've created a Gitter channel while ago, but during the times when we haven't got contributors and it was mostly dead channel, so we've abandoned to use it. If more people would like to join to such channel, we're ready to open it again and mainance.

**Comment by dborovsky** [comment]
Yes. This this good idea. because i would like to continue help with this project and some time i have questions.

**Comment by dborovsky** [comment]
yes. i think. its will very good idea. because I would continue to help with this project. And it`s will be very good idea to comunicate more deep.

**Comment by Leonardo-Ferreira** [comment]
I believe slack is excellent tool for the development team to communicate... quick small questions with no future relevance are not a good fit for forums in my opinion. Specially for alignment regarding pull requests and other time-sensitive matters...
Slack could also be a tool for better troubleshooting with customers... I liked it very much when I had to seek technical help with [Minio](https://min.io)... threads really help!

**Comment by Leonardo-Ferreira** [comment]
OK, so, I create a slack.
Here is the link: https://join.slack.com/t/grandnodeworkspace/shared_invite/enQtNzY3MTcyOTI2NDk4LWU3NGYzZmNhNWY3NjgwMTRkMTg3MzY4MDViY2E3ZGJmMWE0ZDQ0NjFjOGZlNWNmZWVmNDZlYWNhNjM5NmI5ZjA

this is permanent and should be a open link, meaning, click, register and you're on

**Comment by wwrighter** [comment]
No love for Microsoft Teams. I see how it is.

**Comment by Leonardo-Ferreira** [comment]
well... I kinda use them both professionally on a daily basis... I must say I prefer slack because of the "Threads" feature and the broader number of apps/integrations... but, it's the team call... if you guys say that we will use ICQ or MSN Messenger, ill follow along

**Comment by Leonardo-Ferreira** [comment]
@wwrighter + @dborovsky pls join us!
